static/app/components/layouts/thirds.tsx

```diff
@@ -142,11 +142,43 @@ export const Body = styled('div')<{noRowGap?: boolean}>`
   }
 `;
 
+interface MainProps extends React.ComponentProps<typeof MainSection> {
```

Comment:
Nit, but we can probably remove the section element for container, and drop the type indireciton
```suggestion
interface MainProps extends React.HTMLAttributes<HTMLElement> {
```